modules/scicos_blocks/src/c/cmatview.c

```diff
@@ -198,13 +198,14 @@ SCICOS_BLOCKS_IMPEXP void cmatview(scicos_block * block, scicos_flag flag)
             */
             alpha = block->rpar[0];
             beta = block->rpar[1];
-            fprintf(filePointer, "%d %d || %d | %s | %d | %d | %d | %d ||", block_id, processId, iFigureUID, block->uid, iAxeUID, iGrayplotUID, m, n);
+            fprintf(filePointer, "%d %d || %d | %d | %d | %s | %d | %d ||",
+            block_id, processId, iFigureUID, iAxeUID, iGrayplotUID,block->uid, m, n);
```

Comment:
```suggestion
                block_id, processId,
                iFigureUID, iAxeUID, iGrayplotUID,block->uid, m, n);
```
put newline before iFigureUID also